diagnostic.md

```diff
@@ -27,6 +30,11 @@ Record your responses inside the fenced code blocks below each question.
 
     ```html
     <!-- your response here -->
+    {{#each model as |eachcontact| }}
+      {{my-contact  contact=eachcontact}}
+
+    {{/each}}
```

Comment:
If you want to use different binding names then you would need to reference it like this:
```
{{#each model as | eachContact |}}
{{my-contact contact=eachContact.contact}}
{{/each}}
```